apps/frontend/src/pages/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,131 @@
+import { Button, Loader, ProgressBar } from "~/libs/components/components.js";
+import { PREVIOUS_INDEX_OFFSET } from "~/libs/constants/constants.js";
+import {
+	useAppDispatch,
+	useAppForm,
+	useAppSelector,
+	useCallback,
+	useEffect,
+} from "~/libs/hooks/hooks.js";
+import { notification } from "~/libs/modules/notification/notification.js";
+import {
+	actions as onboardingActions,
+	oneAnswerSelectedValidationSchema,
+} from "~/modules/onboarding/onboarding.js";
+import { ONBOARDING_FORM_DEFAULT_VALUES } from "~/pages/onboarding/libs/constants/constants.js";
+
+import { OnboardingAnswer } from "./libs/components/onboarding-answer/onboarding-answer.js";
+import { OnboardingValidationMessage } from "./libs/enums/emuns.js";
+import { type OnboardingFormValues } from "./libs/types/types.js";
+import styles from "./styles.module.css";
+
+const Onboarding: React.FC = () => {
+	const dispatch = useAppDispatch();
+
+	const {
+		currentQuestionIndex,
+		isLastQuestion,
+		question,
+		totalQuestionsAmount,
+	} = useAppSelector(({ onboarding }) => ({
+		currentQuestionIndex: onboarding.currentQuestionIndex,
+		isLastQuestion:
+			onboarding.currentQuestionIndex ===
+			onboarding.questions.length - PREVIOUS_INDEX_OFFSET,
+		question: onboarding.currentQuestion,
+		totalQuestionsAmount: onboarding.questions.length,
+	}));
+
+	useEffect(() => {
+		void dispatch(onboardingActions.getAll());
+	}, [dispatch]);
+
+	const { control, handleSubmit, isValid, reset } =
+		useAppForm<OnboardingFormValues>({
+			defaultValues: ONBOARDING_FORM_DEFAULT_VALUES,
+			validationSchema: oneAnswerSelectedValidationSchema,
```

Comment:
Why 1 answer? We need a validation for a full form

Reply:
This validation is for the full form. 
I can't add Yuliia validation because form doesn't store answers as an array 
`answerIds: []`